datasets/Lab41-SRI-VOiCES.yaml

```diff
@@ -0,0 +1,27 @@
+Name: Voices Obscured in Complex Enrivonmental Settings (VOiCES)
+Description: VOiCES is a speech corpus recorded in acoustically challenging settings,
```

Comment:
Can you change this to a multiline text block like in https://github.com/awslabs/open-data-registry/blob/master/datasets/noaa-nwm-pds.yaml?